### PR TITLE
scripts: check rustc version

### PR DESCRIPTION
**Pull request by overvenus**
Ref #29

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

File: scripts/check_requirement.sh [hunk 1]
```diff
@@ -78,8 +78,8 @@ function install_gpp {
 }
 
 # Check rust
-if which cargo &>/dev/null; then
-    if ! cargo --version | grep nightly &>/dev/null; then
+if which rustc &>/dev/null; then
+    if ! rustc --version | grep nightly &>/dev/null; then
         echo "Please upgrade Rust to nightly."
```
Comment (ngaut) [inline scripts/check_requirement.sh]: Maybe we can display the correct way to upgrade to rust nightly

**Comment by overvenus** [comment]
PTAL @ngaut

**Comment by ngaut** [comment]
LGTM

**Pull request merged**